tests/utils.py

```diff
@@ -175,7 +175,7 @@ def cmp_exprs(left, right):
 
 
 def assert_code_equal(actual, expected):
-    actual.strip() == expected.strip()
+    assert actual.strip() == expected.strip()
```

Comment:
I'm so sorry about this! API test was broken in my PR #193.

Reply:
Oh shoot. My bad, I completely overlooked this 🤦 Thanks for catching and fixing